src/main.ts

```diff
@@ -140,6 +141,13 @@ async function main() {
 				if (response) {
 					message.reply(response);
 				}
+
+				const spamPrevention = await messageIsSpam(message);
+				if (spamPrevention) {
+					message.delete();
+					message.member?.timeout(config.options.timeoutTime, config.messages.timeoutSpamReason);
+					(client.channels.cache.get(config.channels.modChannel) as TextChannel).send("User <@"+message.author.id+"> Has send more than "+config.options.maxMentions+" mentions and therefore has been timed out for "+config.options.timeoutTime+" milliseconds.");
```

Comment:
what is the type this returns? why does it need the cast?

Reply:
TS complains that it can be null.

Reply:
if it can be null, save it to a const variable and wrap the send line in an if